src/main/java/it/gov/pagopa/fdr/controller/middleware/filter/RequestFilter.java

```diff
@@ -89,24 +104,24 @@ public void filter(ContainerRequestContext containerRequestContext) throws IOExc
             .collect(Collectors.joining("\n"));
     containerRequestContext.setEntityStream(new ByteArrayInputStream(body.getBytes()));
 
-    reService.sendEvent(
-        ReInterface.builder()
-            .serviceIdentifier(AppVersionEnum.FDR003)
-            .created(Instant.now())
-            .sessionId(sessionId)
-            .eventType(EventTypeEnum.INTERFACE)
-            .httpType(HttpTypeEnum.REQ)
-            .httpMethod(requestMethod)
-            .httpUrl(requestPath)
-            .payload(body)
-            .header(
-                containerRequestContext.getHeaders().entrySet().stream()
-                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)))
-            .pspId(pspPathParam)
-            .fdr(fdrPathParam)
-            .organizationId(ecPathParam)
-            .fdrAction(fdrActionEnum)
-            .build());
+    /*reService.sendEvent(
```

Comment:
remove this unused code

Reply:
This, as the other parts, is temporarly commented because in next commit will be redefined and maybe this structure can be re-used.